db/schema.rb

```diff
@@ -10,11 +10,91 @@
+    t.string "name", limit: 150, null: false
+    t.index ["name"], name: "auth_group_name_a6ea08ec_like", opclass: :varchar_pattern_ops
+    t.index ["name"], name: "auth_group_name_key", unique: true
+  end
+
+  create_table "auth_group_permissions", force: :cascade do |t|
+    t.integer "group_id", null: false
+    t.integer "permission_id", null: false
+    t.index ["group_id", "permission_id"], name: "auth_group_permissions_group_id_permission_id_0cd325b0_uniq", unique: true
+    t.index ["group_id"], name: "auth_group_permissions_group_id_b120cbf9"
+    t.index ["permission_id"], name: "auth_group_permissions_permission_id_84c5c92e"
+  end
+
+  create_table "auth_permission", id: :serial, force: :cascade do |t|
+    t.string "name", limit: 255, null: false
+    t.integer "content_type_id", null: false
+    t.string "codename", limit: 100, null: false
+    t.index ["content_type_id", "codename"], name: "auth_permission_content_type_id_codename_01ab375a_uniq", unique: true
+    t.index ["content_type_id"], name: "auth_permission_content_type_id_2f476e4b"
+  end
+
+  create_table "auth_user", id: :serial, force: :cascade do |t|
+    t.string "password", limit: 128, null: false
+    t.timestamptz "last_login"
+    t.boolean "is_superuser", null: false
+    t.string "username", limit: 150, null: false
+    t.string "first_name", limit: 150, null: false
+    t.string "last_name", limit: 150, null: false
+    t.string "email", limit: 254, null: false
+    t.boolean "is_staff", null: false
+    t.boolean "is_active", null: false
+    t.timestamptz "date_joined", null: false
+    t.index ["username"], name: "auth_user_username_6821ab7c_like", opclass: :varchar_pattern_ops
+    t.index ["username"], name: "auth_user_username_key", unique: true
+  end
+
+  create_table "auth_user_groups", force: :cascade do |t|
+    t.integer "user_id", null: false
+    t.integer "group_id", null: false
+    t.index ["group_id"], name: "auth_user_groups_group_id_97559544"
+    t.index ["user_id", "group_id"], name: "auth_user_groups_user_id_group_id_94350c0c_uniq", unique: true
+    t.index ["user_id"], name: "auth_user_groups_user_id_6a12ed8b"
+  end
+
+  create_table "auth_user_user_permissions", force: :cascade do |t|
+    t.integer "user_id", null: false
+    t.integer "permission_id", null: false
+    t.index ["permission_id"], name: "auth_user_user_permissions_permission_id_1fbb5f2c"
+    t.index ["user_id", "permission_id"], name: "auth_user_user_permissions_user_id_permission_id_14a6b632_uniq", unique: true
+    t.index ["user_id"], name: "auth_user_user_permissions_user_id_a95ead1b"
+  end
+
+  create_table "authtoken_token", primary_key: "key", id: { type: :string, limit: 40 }, force: :cascade do |t|
+    t.timestamptz "created", null: false
+    t.integer "user_id", null: false
+    t.index ["key"], name: "authtoken_token_key_10f0b77e_like", opclass: :varchar_pattern_ops
+    t.index ["user_id"], name: "authtoken_token_user_id_key", unique: true
+  end
+
```

Comment:
This is a big schema change that is not related to this PR. Let's drop this so that our schema file stays clean 🧹